src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -298,47 +314,51 @@ FROM <repository>/<image>:<tag>
 FROM my.registry.url:9000/image:latest
 ----
 
-Read more about registries in the https://docs.docker.com/registry/[Docker documentation].
+To read more about registries, see the https://docs.docker.com/registry/[Docker documentation].
 
 [id="privileged-mode"]
-== Privileged Mode
+== Privileged mode
 
-Docker privileged mode allows a container to have root-level access to the host from within the container. This enables containers to do more than they normally would, but opens up security risks. You can enable your workspaces to have privileged mode, giving your users root-level access to the host where Che is running (in addition to root access of their workspace). Privileged mode is necessary if you want to enable certain features such as Docker in Docker.
+Docker privileged mode allows a container to have root-level access to the host from within the container. This enables containers to do more than they normally can but also presents security risks. You can enable your workspaces to have privileged mode, giving your users root-level access to the host where Che is running (in addition to root access of their workspaces). Privileged mode is necessary if you want to enable certain features such as Docker in Docker.
 
-By default, Che workspaces powered by a Docker container are not configured with Docker privileged mode. There are many security risks to activating this feature - please review the various issues with blogs posted online.
+By default, Che workspaces powered by a Docker container are not configured with Docker privileged mode. There are many security risks to activating this feature. Review the various issues with blogs posted online.
```

Comment:
What blogs? If I'm reading this, I probably don't know what security risks there are.

Reply:
Created at https://github.com/eclipse/che-docs/issues/560.